### PR TITLE
Add projects and about sections

### PR DESCRIPTION
**Pull request by yersel500**
I added:
- Four projects section
-  About me section

I used:
- html
- CSS (flexbox, grid)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -41,6 +41,132 @@ <h2>LET'S CONNECT</h2>
       
     </main>
   </div>
+
+    <div class="project-wrap">
```
Comment (Cathella) [inline index.html]: Hi @yersel500 , I am impressed by the great effort you have invested in this project. I looks ⭐ beautiful ⭐. However we got a few changes to make 🤭.
- [ ] I think the structure of the projects section has been interchanged and could be the other way round according to this project requirement 👇🏻, while we make the children `<div>` tags

image

File: index.html [hunk 2]
```diff
@@ -41,6 +41,132 @@ <h2>LET'S CONNECT</h2>
...
+            <li>javaScript</li>
+          </ul>
+          <div class="button">
+            <button class="button1" type="button">See Project</button>
+          </div>
+        </div>
+      </section>
+    </div>
+
+    <div class="wrapper2">
```
Comment (Cathella) [inline index.html]: - [ ] As mentioned above please make the container of the about section is a `<section>` tag. This is helpful in proving our code readability 😄

File: index.html [hunk 3]
```diff
@@ -41,6 +41,132 @@ <h2>LET'S CONNECT</h2>
...
+      <div class="second">
+        <h4>About<br>
+          Myself
+        </h4>
+        <p>
+          Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <h2>LET'S CONNECT</h2>
```
Comment (Cathella) [inline index.html]: - [ ] I think you are confusing the heading hierarchy here. I believe `<h2>LET'S CONNECT</h2>` is meant to be wrapped in an `<h4>` tag instead while the ` <h4>About<br>Myself</h4>` takes the `<h2>` tag. 

❓ What do you think is the more important tag 🤷🏻

**Review by Cathella — CHANGES_REQUESTED** [review]
## 🟨 Status: Changes Required 🛑

### Hi @yersel500 👋🏻, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## The Good points 💪🏻

✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ The sections use the font and colors defined for the template
✔️ Used Flexbox to place elements in the page.
✔️ Used CSS Grid to place elements in the page.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kingstalux — APPROVED** [review]
Hi @yersel500 , 
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

Good job implementing the required changes. 👍 Also I'd like to also acknowledge the fact that you've added a functioning live demo link to your README file 💯 . Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**